File: Utilities/DatabaseUpdater/DatabaseProjectScriptSourceTests.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting
<TestClass()> _
Public Class DatabaseProjectScriptSourceTests
    <TestMethod()> _
    Sub GoUpALevel_DummyData_Test()
        Dim dp As New DatabaseProjectScriptSource("asds")
        dp.currentPath = ".\hello\"
        dp.GoUpALevel()
        Assert.AreEqual(".\", dp.currentPath)
    End Sub


End Class

File: Utilities/SqlScriptRunner/ScriptSources/DatabaseProjectScriptSourceTests.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using NUnit.Framework;
namespace SqlScriptRunner.ScriptSources
{
    
    [TestFixture]
    public class DatabaseProjectScriptSourceTests
    {

        [Test]
        public void GoUpALevel_DummyData_Test()
        {
            DatabaseProjectScriptSource dp = new DatabaseProjectScriptSource("asds");
            dp.currentPath = ".\\hello\\";
            dp.GoUpALevel();
            Assert.AreEqual(".\\", dp.currentPath);
        }
    }
}
